server/src/main/java/org/elasticsearch/indices/recovery/RecoverySettings.java

```diff
@@ -384,21 +612,6 @@ public void setInternalActionLongTimeout(TimeValue internalActionLongTimeout) {
         this.internalActionLongTimeout = internalActionLongTimeout;
     }
 
-    private void setMaxBytesPerSec(ByteSizeValue maxBytesPerSec) {
```

Comment:
nit: I think this getter/setter just got moved too, could we move it back to clarify it was unchanged?

Reply:
Sure, I pushed 99f7109f4b11c6afa787040bbb5f6cc3ea9a5c9d